### PR TITLE
Bump milli to v0.37.2

### PR DESCRIPTION
**Pull request by curquiza**
⚠️ Please ensure milli has been updated in EVERY cargo.tom ⚠️

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1351,8 +1351,8 @@ dependencies = [
 
 [[package]]
 name = "flatten-serde-json"
-version = "0.37.1"
-source = "git+https://github.com/meilisearch/milli.git?tag=v0.37.1#e1b2113cd9a467410841a254286e5e25961af43e"
+version = "0.37.2"
+source = "git+https://github.com/meilisearch/milli.git?tag=v0.37.2#1582b96119fedad39c726a6d4aeda0f53e868a3b"
```
Comment (Kerollmops) [inline Cargo.lock:1355]: Not sure why but [the commit looks aligned with the v0.37.2 release](https://github.com/meilisearch/milli/releases/tag/v0.37.2) but why is it not the same as [the last commit of the main branch](https://github.com/meilisearch/milli/commits/main)?

Comment (curquiza) [reply Cargo.lock:1355]: v0.37.2 of milli is not done on main, but on the `release-v0.37.2` branch, I wrote it on the release description 😇 
https://github.com/meilisearch/milli/releases/tag/v0.37.2

Comment (curquiza) [reply Cargo.lock:1355]: Again, there are too many un-wanted PRs on `main` for v0.30.2 release

Comment (curquiza) [reply Cargo.lock:1355]: You should check [this branch instead](https://github.com/meilisearch/milli/tree/release-v0.37.2)

Comment (Kerollmops) [reply Cargo.lock:1355]: Whooops, indeed, therefore it looks good to me 👌

**Review by Kerollmops — APPROVED** [review]
bors merge

**Pull request merged**